package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=7.0.0"
   },
   "dependencies": {
+    "@ltd/j-toml": "1.38.0",
```

Comment:
I'm not sure we actually need this. Sorry for not seeing this before and pointing it out but the Insights service already supports importing on-prem blueprints and I think there's already a "toml" package we can use.

Reply:
I know it's @croissanne's last day before PTO, but maybe she has some suggestions

Reply:
This PR https://github.com/osbuild/image-builder/pull/1442 needs to land first before I can fully add this